packages/debian/Dockerfile

```diff
@@ -11,6 +11,7 @@ RUN apt update -y \
         devscripts \
         build-essential \
         lintian \
+	apt-rdepends \
```

Comment:
wrong indentation (I bet your editor added a tab, whereas the other lines are using spaces)

Reply:
@jbertran made me the same remark but there is no tindentation issue in the file in my computeur and neither on the files on github (see clicking on `view file`).

Reply:
Likely because we indent with 4 spaces and by default tabs are displayed with a 4-spaces width so visually it may look OK.

But it's still bad to mix tab and spaces, and this will be quite ugly if you editor use a custom width for the tabs and/or display tab differently from spaces.

Reply:
Indeed, I did a tabulation and not spaces.
It's corrected !